setup.py

```diff
@@ -35,31 +34,34 @@
 
 PY3 = sys.version_info[0] == 3
 
-with io.open('README.md', encoding='utf-8') as f:
-    long_description = f.read()
+try:
+    FileNotFoundError
+except NameError:
+    FileNotFoundError = IOError
 
+try:
+    with io.open('README.md', encoding='utf-8') as f:
+        long_description = f.read()
+except FileNotFoundError:
+    long_description = ''
 
-class Tox(TestCommand):
-    user_options = [('tox-args=', None, "Arguments to pass to tox")]
+
+class CleanCommand(Command):
+    """Custom clean command to tidy up the project root."""
```

Comment:
how do you think to remove `Custom clean` and rephrase:
`Command to tidy up the project root.`

Reply:
```
In [6]: help(CleanCommand)
Help on class CleanCommand in module __main__:

class CleanCommand(Command)
 |  Command to tidy up the project root.
 |
 |  Method resolution order:
```

Reply:
Fine for me. I just did not want to change this too much, but Now I am going to split this a bit more into several small PRs so maybe such fixes make more sense :)

Reply:
Fixed.